kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -415,6 +416,10 @@ export class UnityInterface {
     this.gameInstance.SendMessage('Bridges', 'UpdateRealmsInfo', JSON.stringify(realmsInfo))
   }
 
+  public SendPublishSceneResult(result: DeploymentResult) {
+    this.gameInstance.SendMessage('Main', 'PublishSceneResult', JSON.stringify(result))
```

Comment:
This method is already present on Unity side? Or we are going to add it via another PR?

Reply:
We will need to add it on another PR